info.urbanterror.UrbanTerror.yaml

```diff
@@ -0,0 +1,58 @@
+app-id: info.urbanterror.UrbanTerror
+runtime: org.freedesktop.Platform
+runtime-version: '21.08'
+sdk: org.freedesktop.Sdk
+command: urbanterror
+rename-icon: urbanterror
+rename-desktop-file: urbanterror.desktop
+rename-appdata-file: urbanterror.appdata.xml
+finish-args:
+  - --socket=pulseaudio
+  - --socket=x11
+  - --share=ipc
+  - --share=network
+  - --device=dri
+  - --filesystem=~/.q3a/q3ut4/
```

Comment:
Any reason this is `--filesystem` and not `--persist=.q3a`?

Reply:
Thanks. That fixes the game not saving configuration (asking for a player name on every start)